src/platforms/stlinkv3/Readme.md

```diff
@@ -0,0 +1,54 @@
+# Blackmagic firmware for STLINK-V3 Adapters
```

Comment:
Let's name the file `README.md` for consistency with the rest of the repo readme files.

Uwe had a habit of using all-caps names with his own hyphenation for the names of tools, which causes user confusion - ST uses `ST-Link v3` as the official name of the tool, we should to, let's change that throughout this readme for consistency.

Reply:
I actually think that STLINK-V3 is the correct stylising of the name here. See https://www.st.com/en/development-tools/stlink-v3set.html. Will rename the file for consistency.

Reply:
They have STLINK-V3SET and they also have ST-Link v3 it seems.. unless we're missing something.. at the very least if you're right that would make it inconsistent with their previous ST-Link tooling which is.. don't like.. we'll noodle on that some more then